alignak_module_backend/arbiter/module.py

```diff
@@ -290,12 +297,16 @@ def get_realms(self):
         :return: None
         """
         self.configraw['realms'] = {}
+        self.configraw['realms_name'] = {}
         all_realms = self.backend.get_all('realm', {'embedded': json.dumps({'_children': 1})})
         logger.info("Got %d realms",
                     len(all_realms['_items']))
         for realm in all_realms['_items']:
             logger.info("- %s", realm['name'])
             self.configraw['realms'][realm['_id']] = realm['name']
+            self.configraw['realms_name'][realm['name']] = realm['_id']
+            if realm['_level'] < self.highlevelrealm['level']:
```

Comment:
Because arbiter not have realm information, so I get the most higher level realm